### PR TITLE
White-label migration guide

### PR DESCRIPTION
**Pull request by chadwcarlson**
## Why

Closes https://github.com/platformsh/platformsh-docs/issues/3347

## What's changed

Continues https://github.com/platformsh/platformsh-docs/pull/3337. Adding a migration guide for white label partners migrating from Platform.sh as an example.

### CODE REVIEW

**Comment by flovntp** [comment]
@chadwcarlson 
strange that the generated url for updated pages is having `src/` in it, it should be 
https://vendor-3347-migrate-nmrndqi-652soceglkw4u.eu-3.platformsh.site/projects/region-migration.html

**Comment by flovntp** [comment]
Still have an issue in the menu on the right for existing sections.
Capture d’écran 2023-08-16 à 10 36 52

**Comment by chadwcarlson** [comment]
@flovntp Try replacing

```
### {{< vendor/name >}}
```

with

```
### {{% vendor/name %}}
```

**Comment by flovntp** [comment]
@chadwcarlson 
i write a [section in the README file](https://github.com/platformsh/platformsh-docs/tree/vendor/3347-migrate#whitelabel-management) for adding a new whitelabel site (needs to be completed!) and i replace usage of placeholder using `{{< vendor/... >}}` by `{{% vendor/... %}}` in all the file using it (maybe i missed some?). 
And i didn't advise using `{{< vendor/... >}}` in the README section, just a note telling that if using it, they will have issue in the table of contents.

**Review by chadwcarlson — COMMENTED** [review]
Added some reorganization around the current "Learn" structure.

- For the guide, there seems to be an awful lot included on this single page. Could we split the "what you need to know" into it's own page, and then the actual migration steps in another? Is there too much background here that could be put elsewhere in the docs?
- This guide still needs updates for the final step (push/deploy) as well as updates to the `resources` commands.

**Comment by gilzow** [comment]
The [/projects/region-migration.html page at step 2](https://docs.upsun.com.vendor-3347-migrate-nmrndqi-652soceglkw4u.eu-3.platformsh.site/projects/region-migration.html#2-create-a-new-project) mentions choosing a development plan, but those plans aren't applicable to Upsun (see #3452 ). 

> start with a Development plan and upsize it

Also, [lower in the page at step 8 ](https://docs.upsun.com.vendor-3347-migrate-nmrndqi-652soceglkw4u.eu-3.platformsh.site/projects/region-migration.html#8-switch-to-the-new-site)it mentions  picking the right "plan size" so we'll need to update this to `container_profile` ?

File: sites/friday/src/learn/tutorials/migrating/platformsh.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+{{< note >}}
+
+Make sure you remove the deprecated `size`, `disk` and `resources` keys from your {{% vendor/name %}} YAML configuration.
+
+{{< /note >}}
+
+[//]: # (TODO change it to doc page link)
+List of profile_size can be found [here](https://docs.google.com/presentation/d/1PxDJm9c3OCjZxlI2keGy6V7BlAZJGfyueQ1D117YCns/edit?pli=1#slide=id.g20fc593a519_0_32)
+
+{{< /codetabs >}}
```
Comment (AnouckColson) [inline sites/friday/src/learn/tutorials/migrating/platformsh.md:426]:
```suggestion
Make sure you configure resources (CPU, RAM, disk) per environment for each of your applications and services.
For more information, see how to [manage resources](/manage-resources.md).
```

File: sites/friday/src/learn/tutorials/migrating/platformsh.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+
+```bash
+{{% vendor/cli %}} deploy
+```
+
+{{< note title="Warning" >}}
+
+First deployment should fail if you yet didn't allocate resources to your application.
+
+To allocate resources to your application, please see [Resources Allocation doc page](#4-define-resources)
```
Comment (AnouckColson) [inline sites/friday/src/learn/tutorials/migrating/platformsh.md:441]:
```suggestion
For more information on how you can do so, see how to [manage resources](/manage-resources.md).
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**